### PR TITLE
[release-2.3] prow jobs fix

### PR DESCRIPTION
**Pull request by dislbenn**
Issue: https://github.com/stolostron/backlog/issues/18729

### CODE REVIEW

File: sonar-project.properties
Comment (SherinV) [inline sonar-project.properties:1]: `sonar.projectKey=stolostron_search-e2e-test`
```suggestion
sonar.projectKey=stolostron_search-e2e-test
```

Comment (SherinV) [reply sonar-project.properties:1]: Some of our new projects, including search-v2-api, had stolostron_. But e2e test seems to be open-cluster-management_*
image

**Comment by dislbenn** [comment]
Splitting up PR for org rename and prow work:
- https://github.com/stolostron/search-e2e-test/pull/179

**Comment by dislbenn** [comment]
The next step for this PR is to re-enable the e2e test; however, I will start only with the API test.

File: start-tests.sh
Comment (jlpadilla) [inline start-tests.sh:147]: I think you have had some problems with pods not coming up before, so maybe we can add some logic to confirm that everything we need is running.  Narrow it down to the pods we need to avoid failing when other components have problems.

```
SEARCH_RUNNING_COUNT=${oc get pod -n open-cluster-management | grep search- |grep Running | wc -l)
```
```
CONSOLE_RUNNING_COUNT=$(oc get pod -n open-cluster-management | grep search- |grep Running | wc -l)
```

Then log a message and maybe wait longer if the count doesn't match what we expect.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Please see comment before merging. It could be added in a separate PR as an enhancement.

File: start-tests.sh
Comment (dislbenn) [reply start-tests.sh:147]: I agree with this solution. I will create a later PR adding this as an enhancement.

**Pull request merged**
